### PR TITLE
chore(con-372): updating deprecated set-output command

### PR DESCRIPTION
**Pull request by mtktran**
- Set-output command is being deprecated in favor of using github enviroment files
- PR updates actions/core library and updates scripts to use github environment files

### CODE REVIEW

**Review by mtktran — COMMENTED** [review]
Tested by making changes to helmfile and terraform here are the following tests: 
https://github.com/iStreamPlanet/kubernetes_management/actions/runs/3605648696/jobs/6076245551
https://github.com/iStreamPlanet/kubernetes_management/actions/runs/3605648702/jobs/6076245601

File: .github/workflows/lint-pr.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     name: Validate PR Title
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v4.6.0
+      - uses: amannn/action-semantic-pull-request@v5.0.2
```
Comment (glichtenISP) [inline .github/workflows/lint-pr.yml:16]: This update seems to be breaking the linter?

**Review by glichtenISP — CHANGES_REQUESTED** [review]

File: .github/workflows/lint-pr.yml (hunk 1 above)
Comment (mtktran) [reply .github/workflows/lint-pr.yml:16]: @glichtenISP This has to be updated because it is also using the old set-output command

**Review by glichtenISP — APPROVED** [review]
Not sure why the linter is failing, but LGTM

**Pull request merged**